google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionImpl.java

```diff
@@ -597,4 +594,12 @@ public void setRequestIdCreator(XGoogSpannerRequestId.RequestIdCreator creator)
   public XGoogSpannerRequestId.RequestIdCreator getRequestIdCreator() {
     return this.requestIdCreator;
   }
+
+  int getChannel() {
+    if (getIsMultiplexed()) {
+      return 0;
+    }
```

Comment:
I assume that this means that we will be getting the channel that is being used in a different way for multiplexed sessions, and that this method is only intended for actual use with regular sessions.